src-cpp/server42/src/Workers.cpp

```diff
@@ -64,8 +77,13 @@ namespace org::openkilda {
                             mbuf_send_buffer[i].initFromRawPacket(*(pos+i), device);
                             mbuf_send_buffer_p[i] = &mbuf_send_buffer[i];
                             mbuf_send_buffer_p[i]->setFreeMbuf(true);
-                        }
 
+                            //TODO: can be switched to raw memory arithmetics
+                            pcpp::Packet parsed_packet(mbuf_send_buffer_p[i], false, pcpp::UDP);
+                            auto *udp_layer = parsed_packet.getLayerOfType<pcpp::UdpLayer>();
+                            auto *payload = reinterpret_cast<org::openkilda::Payload *>(udp_layer->getLayerPayload());
+                            payload->t0 = htobe64(timestamp);
```

Comment:
`htobe64` can/shoult be done outside this(all) loop.

Reply:
done